### PR TITLE
add sifa123

### LINKED ISSUE #410: Sefat Siddiquea
Screenshot (48)

### PR DESCRIPTION
**Pull request by sifa123**
resolve #410

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -694,5 +694,11 @@
     "githubUsername": "ChinmayMhatre",
     "imageUrl": "https://user-images.githubusercontent.com/51131670/111893373-5091f600-8a28-11eb-8e94-48b23eec0495.PNG",
     "issueNumber": 403
-  }
+  },
+   {
```
Comment (AllanRegush) [inline data.json:698]: This bracket should line up like the rest.
```suggestion
  {
```

File: data.json [hunk 2]
```diff
@@ -694,5 +694,11 @@
...
+    "name": "Sefat Siddiquea Sifa",
+    "githubUsername": "sifa123",
+    "imageUrl": "https://user-images.githubusercontent.com/63462931/112490528-a3add500-8da9-11eb-853e-9972b42e5caf.png",
+    "issueNumber": 410
+  },
```
Comment (AllanRegush) [inline data.json:703]: There should be no comma here.  As Per [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas#trailing_commas_in_json). 
```suggestion
  }
```

**Review by AllanRegush — CHANGES_REQUESTED** [review]
Looking Good :+1: Just some minor adjustments :nerd_face:

**Comment by sifa123** [comment]
@AllanRegush Thanks. I have fixed it

**Review by galahad42 — APPROVED** [review]

File: data.json (hunk 1 above)
Comment (AllanRegush) [reply data.json:698]: Could you move the bracket so it lines up with the rest. Just to make the style consistant. :+1:

Comment (sifa123) [reply data.json]: Done

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank you

**Pull request merged**